internal/config/authentication.go

```diff
@@ -314,7 +329,8 @@ type AuthenticationMethodTokenBootstrapConfig struct {
 // AuthenticationMethodOIDCConfig configures the OIDC authentication method.
 // This method can be used to establish browser based sessions.
 type AuthenticationMethodOIDCConfig struct {
-	Providers map[string]AuthenticationMethodOIDCProvider `json:"providers,omitempty" mapstructure:"providers"`
+	EmailMatches string                                      `json:"emailMatches,omitempty" mapstructure:"email_matches"`
```

Comment:
i think this should be a string slice like https://github.com/flipt-io/flipt/blob/2416a0149f1275db5d201ef5d8b8316e01c95b3c/internal/config/cors.go#L12

Reply:
@markphelps Yeah let me change that up. I had it as that initially, had trouble with some testing, but its all good now.